packages/relay/src/lib/eth.ts

```diff
@@ -146,17 +146,17 @@ export class EthImpl implements Eth {
    * @param chain
    */
   constructor(
-    nodeClient: SDKClient,
+    clientSevice: ClientService,
     mirrorNodeClient: MirrorNodeClient,
     logger: Logger,
     chain: string,
     cache?
   ) {
-    this.sdkClient = nodeClient;
+    this.clientService = clientSevice;
```

Comment:
nit: typo `clientSevice` should be `clientService`

Reply:
Fixed.